requirements.txt

```diff
@@ -2,6 +2,8 @@ requests
 pandas
 influxdb
 urllib3
+ujson
+logger
```

Comment:
The `logger` is variable from `logging` module, not required module.

Reply:
`logger` will be removed